CHANGELOG/CHANGELOG-1.12.md

```diff
@@ -1,5 +1,16 @@
 # Changelog 1.12
 
+## 1.12.1 - 2023-06-27
+
+### Fixed
+
+- Bump up libOpenflow and ofnet versions to fix a PackintIn2 response parse error. ([#5154](https://github.com/antrea-io/antrea/pull/5154), [@wenyingd])
+- Fix incorrect FlowMod message passing in openflow client modifyFlows to avoid unexpected flow error. ([#5125](https://github.com/antrea-io/antrea/pull/5125), [@Dyanngg])
```

Comment:
in the modifyFlows function of the OpenFlow client

Reply:
Done